Makefile

```diff
@@ -9,6 +9,7 @@ lint: tools/go.mod
 	go run -modfile=tools/go.mod honnef.co/go/tools/cmd/staticcheck -checks=all ./...
 	go list -m -json $(MODULE_DEPS) | go run -modfile=tools/go.mod go.elastic.co/go-licence-detector \
 		-includeIndirect -rules tools/notice/rules.json -validate
+	go mod tidy
```

Comment:
```suggestion
	go mod tidy && git diff --exit-code
```

Reply:
I didn't include it since [git status --porcelain](https://github.com/elastic/apm-queue/blob/681d564ac14c318446a7441d143be6e151fe8de1/.github/workflows/ci.yml#L18-L24) was part of the GitHub action itself.

But I think it's a good idea so it's agnostic to the CI pipeline